docker-compose.yaml

```diff
@@ -37,8 +37,9 @@ services:
     ports:
       - "9091:8080"
     healthcheck:
-      test: [ "CMD", "curl", "-f", "http://localhost:9091/health" ]
+      test: [ "CMD", "curl", "-f", "http://localhost:8080/health/ping" ]
```

Comment:
[As per the docs](https://docs.docker.com/engine/reference/builder/#healthcheck) using `CMD`
> check container health by running a command inside the container

Therefore we should use the port the app is running on inside the container

Reply:
agreed, but probably makes more sense to carry on using `/health` to ensure it is all healthy rather than just being able to ping the server?